Assets/Plugins/iOS/Shopify/WebCheckoutSession.swift

```diff
@@ -74,3 +78,12 @@ private extension UnityAppController {
         return unityController.rootViewController
     }
 }
+
+// Small helper class for presenting the SFSafariViewController
+private class WebCheckoutNavigationController: UINavigationController {
+        
+    // Only allow the same rotations the underlying game allows.
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        return UnityAppController.root.supportedInterfaceOrientations
+    }
+}
```

Comment:
We should be using this `UINavigationController` for any screens we want to modally present on top of the Unity controller so I think renaming this to something like `UnityModalNavigationController` makes more sense.

Wouldn't hurt to also override those two options!